os_builders/roles/nubes_bootcontext/files/nubes-bootcontext.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
```

Comment:
Could we 
`set -euo pipefail` if we want to fail noisly?

Reply:
Nope, failing noisily may mean that a user cant logon